src/simGCN.c

```diff
@@ -785,9 +785,151 @@ s32 simulatorTestReset(s32 IPL, s32 forceMenu, s32 allowReset, s32 usePreviousSe
 
 #pragma GLOBAL_ASM("asm/non_matchings/simGCN/simulatorMCardPollDrawFormatBar.s")
 
+// matches but data doesn't
+#ifndef NON_MATCHING
 #pragma GLOBAL_ASM("asm/non_matchings/simGCN/simulatorDrawCursor.s")
+#else
+static s32 simulatorDrawCursor(s32 nX, s32 nY) {
+    GXColor color;
+    s32 nTick;
+    u8 var_r5;
+    s32 pad;
+
+    GXClearVtxDesc();
+    GXSetVtxDesc(GX_VA_POS, GX_DIRECT);
+    GXSetVtxAttrFmt(GX_VTXFMT7, GX_VA_POS, GX_POS_XY, GX_S16, 0);
+    GXSetNumChans(1);
+    GXSetNumTexGens(0);
+    GXSetNumTevStages(1);
+    GXSetTevColorIn(GX_TEVSTAGE0, GX_CC_ZERO, GX_CC_ZERO, GX_CC_ZERO, GX_CC_C0);
+    GXSetTevAlphaIn(GX_TEVSTAGE0, GX_CA_ZERO, GX_CA_ZERO, GX_CA_ZERO, GX_CA_A0);
+    GXSetTevColorOp(GX_TEVSTAGE0, GX_TEV_ADD, GX_TB_ZERO, GX_CS_SCALE_1, GX_FALSE, GX_TEVPREV);
+    GXSetTevAlphaOp(GX_TEVSTAGE0, GX_TEV_ADD, GX_TB_ZERO, GX_CS_SCALE_1, GX_FALSE, GX_TEVPREV);
+    GXSetTevOrder(GX_TEVSTAGE0, GX_TEXCOORD_NULL, GX_TEXCOORD_NULL, GX_TEXCOORD_NULL);
+
+    nTick = OSGetTick() >> 14;
+    if (nTick & 0x100) {
+        var_r5 = (u8)nTick ^ 0xFF;
+    } else {
+        var_r5 = (u8)nTick;
+    }
+
+    color.b = 0;
+    color.g = 0;
+    color.r = 0;
```

Comment:
this seems to match too
```suggestion
    color.r = color.g = color.b = 0;
```